tests/test_knowledge_panels.py

```diff
@@ -228,3 +232,67 @@ def test_data_quality_kp_with_all_three_values(monkeypatch):
             ],
         }
     }
+
+
+def test_last_edits_kp_with_all_three_values(monkeypatch):
+    expected_url = "https://hu-en.openfoodfacts.org/api/v2/search"
+    expected_kwargs = {
+        "params": {
+            "fields": "product_name,code,last_editor,last_edit_dates_tags",
+            "sort_by": "last_modified_t",
+            "vitamins_tags_en": "vitamin-k",
+        }
+    }
+    expected_json = {
```

Comment:
we named previous variables because expected_url and expected_kwargs are meant to check effective url and kwargs used.

While the json is more something you are giving to build the test. So I would name it `json_content`